modules/command/src/main/java/org/apache/fluo/command/CommonCommand.java

```diff
@@ -17,18 +17,12 @@
 
 import com.beust.jcommander.Parameter;
 
-class CommonOpts extends ConfigOpts {
+abstract class CommonCommand extends ConfigCommand {
```

Comment:
I think that name is fine. I chose `CommandCommand` to keep it consistent with `CommonOpts`, but I think `AppCommand` may be more descriptive.

On a somewhat related note I was thinking of combing `ConfigCommand` and `BaseCommand` into the `BaseCommand` class because I'm not sure if there's any benefit from separating them out.

Reply:
Is Config command the only thing that extends BaseCommand?  If so it would make sense to collapse them.

Reply:
Yes, `ConfigCommand` is the only thing that extends `BaseCommand`. I'll collapse them.

Reply:
Actually I think FluoExec should extend BaseCommand so that you can pass a -h option. So I'll leave them separate.